ntbs-service/Pages/Notifications/OverviewPartials/_PreviousHistoryOverviewPartial.cshtml

```diff
@@ -20,7 +20,7 @@
     <nhs-grid-row>
         <nhs-grid-column grid-column-width="OneQuarter">
             <div class="notification-details-label"> Previous notification of TB </div>

```

Comment:
While here, should we update this to use the `Display(Name` format?

Reply:
I'm not sure on this one - we aren't doing this in most places and if we start changing here should we change everywhere? Plus it will be a long notification.PatientTbHistories.PreviouslyHadTb chain which looks messy. Happy to make the change if you think it's worth it though!